.github/workflows/codeql.yml

```diff
@@ -25,6 +25,11 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - uses: actions/checkout@v3
```

Comment:
Duplication of "uses", typo?

Reply:
Not typo - it's invoking the Checkout action twice, once for the main repo and again to get a copy of the RDMP source to build against.